frontstage/common/mappers.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+
+import iso8601
+
+
+def convert_events_to_new_format(events):
+    formatted_events = {}
+    for event in events:
+        date_time = iso8601.parse_date(event['timestamp'])
```

Comment:
Need to wrap this bit in a try/except block I think.  If the timestamp isn't correct for whatever reason, it'll throw a ParseError that needs to be handled.

Might be worth testing this with a duff timestamp and see what happens?